Project/Source/Panels/PanelHierarchy.cpp

```diff
@@ -313,6 +336,7 @@ GameObject* PanelHierarchy::CreateUIProgressBar(GameObject* gameObject) {
 	ComponentTransform2D* progressTransform2D = progressBar->CreateComponent<ComponentTransform2D>();
 	ComponentCanvasRenderer* progressRenderer = progressBar->CreateComponent<ComponentCanvasRenderer>();
 	ComponentProgressBar* progress = progressBar->CreateComponent<ComponentProgressBar>();
+	progressBar->Init();
 
 	GameObject* background = CreateUIImage(progressBar);
 	background->GetComponent<ComponentTransform2D>()->SetSize(float2(700, 80));
```

Comment:
Here is not necessary to call the Init()?

Reply:
No, it's not necessary. It gets called inside CreateUIImage.